hadoop-crypto-api/build.gradle

```diff
@@ -7,4 +7,5 @@ dependencies {
     compile "org.slf4j:slf4j-api"
     compile "com.fasterxml.jackson.core:jackson-databind"
     compile "com.google.guava:guava"
+    compile "org.apache.hadoop:hadoop-common"
```

Comment:
can't depend on hadoop-common in the api project. brings in too much that api consumers shouldn't need to depend on.